src/NuGet.Clients/NuGet.PackageManagement.UI/PackageManagerToolWindowPane.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace NuGet.PackageManagement.UI
+{
+    [ProvideToolWindow(typeof(PackageManagerToolWindowPane), Style = VsDockStyle.Tabbed, DocumentLikeTool = true, Window = EnvDTE.Constants.vsCATIDDocument)]
+    public class PackageManagerToolWindowPane : ToolWindowPane, IVsWindowFrameNotify3
+    {
+        private PackageManagerControl _content;
+
+        /// <summary>
+        /// Initializes a new instance of the EditorPane class.
+        /// </summary>
+        public PackageManagerToolWindowPane(PackageManagerControl control)
+            : base(null)
+        {
+            _content = control;
+        }
+
+        public PackageManagerModel Model
+        {
+            get { return _content.Model; }
+        }
+
+        /// -----------------------------------------------------------------------------
+        /// <summary>
+        /// IVsWindowPane
+        /// </summary>
+        /// -----------------------------------------------------------------------------
+        public override object Content
```

Comment:
this xmldoc doesn't seem helpful, plus none of the other public methods have xmldoc, so I think it should just be deleted here.